doc/releases/changelog-dev.md

```diff
@@ -18,6 +18,37 @@
 
 <h3>Improvements</h3>
 
+* `QuantumTape` objects are now iterable and can be used in `for` loops or
```

Comment:
I think important to also mention that we can index into them via `tape[i]`.

Reply:
Sure thing! Updated.